tests/Database/SelectFields/AlwaysRelationTests/UserType.php

```diff
@@ -25,6 +25,10 @@ public function fields(): array
             'name' => [
                 'type' => Type::nonNull(Type::string()),
             ],
+            'posts' => [
+                'type' => Type::listOf(GraphQL::type('Post')),
+                'always' => 'comments',
+            ],
```

Comment:
Sorting

Reply:
alphabetically, or are you talking about something else?

Reply:
Sorry, alphabetically :-)

Reply:
done 👍